requirements.txt

```diff
@@ -57,4 +57,5 @@ tomli==2.0.1
 typing-extensions==4.2.0
 urllib3==1.26.9
 websocket-client==1.3.2
-yarl==1.7.2
```

Comment:
It looks like there was no eol at eof, please check you editor settings and update files if such exist.